output/schema/schema.json

```diff
@@ -100034,7 +100034,7 @@
         {
           "description": "A comma separated list of influencer field names. Typically these can be the by, over, or partition fields that are used in the detector configuration. You might also want to use a field name that is not specifically named in a detector, but is available as part of the input data. When you use multiple detectors, the use of influencers is recommended as it aggregates results for each influencer entity.",
           "name": "influencers",
-          "required": true,
+          "required": false,
```

Comment:
`false` is correct if this interface is only using when putting jobs.  But if it's also used when getting jobs then it needs to be split, because `influencers` will always be present in `analysis_config` when getting jobs.